rosdep/base.yaml

```diff
@@ -1451,6 +1451,18 @@ glpk:
   gentoo: [sci-mathematics/glpk]
   rhel: [glpk-devel]
   ubuntu: [libglpk-dev]
+glslc:
+  arch: [shaderc]
+  debian: [glslc]
```

Comment:
```suggestion
  debian:
    '*': [glslc]
    bullseye: null
```

Reply:
Done in https://github.com/ros/rosdistro/pull/40284/commits/5e286d1de9b0b0d637fe83067964e040278f41d8